client/components/filter-rank.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Select from 'react-select';
+export default class FilterRank extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedOption: null
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange = selectedOption => {
+    this.setState({ selectedOption });
+  };
```

Comment:
If you use this arrow function class property syntax you don't need to also bind the method.